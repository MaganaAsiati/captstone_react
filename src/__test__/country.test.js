import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { HashRouter as Router } from 'react-router-dom';
import store from '../Redux/configureStore';
import Info from '../components/country/country';

describe('info Component test', () => {
  it('Renders Info link', () => {
    const app = renderer
      .create(
        <Provider store={store}>
          <Router>
            <Info />
          </Router>
        </Provider>,
      )
      .toJSON();
    expect(app).toMatchSnapshot();
  });
});
