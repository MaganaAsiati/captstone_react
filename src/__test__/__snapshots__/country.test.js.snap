// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`info Component test Renders Info link 1`] = `
<div
  className="country-details-container"
>
  <div
    className="detail-container"
  >
    <a
      href="#/"
      onClick={[Function]}
    >
      <img
        alt="leftarrow"
        className="back-left-img"
        height={30}
        src="leftarrow.png"
        width={30}
      />
    </a>
    <div
      className="img-container"
    >
      <img
        alt="country-flag"
      />
    </div>
    <h2 />
    <ul
      className="country-details"
    >
      <li>
        The country covid19 cases are :
         
        <span
          className="details"
        />
      </li>
      <li>
        The country number of deaths :
         
        <span
          className="details"
        />
      </li>
      <li>
        The country belongs to
         
        <span
          className="details"
        />
         
        continent
      </li>
    </ul>
    <div
      className="animated-icon"
    >
      <img
        alt="microbe-icon"
        height={50}
        src="iconmicrobe.png"
        width={50}
      />
    </div>
    <div
      className="animated-icon"
    >
      <img
        alt="microbe-icon"
        height={50}
        src="iconmicrobe.png"
        width={50}
      />
    </div>
    <div
      className="animated-icon"
    >
      <img
        alt="microbe-icon"
        height={50}
        src="iconmicrobe.png"
        width={50}
      />
    </div>
  </div>
</div>
`;
